### PR TITLE
Add recommendation to put hyphen after OCID prefix

### LINKED ISSUE #917: Add recommendation to put hyphen after OCID prefix
This is the practice in most implementations (and all examples), but it is not explicit. We can't make it a MUST rule, because that will be backwards-incompatible with at least Zambia.

### PR DESCRIPTION
**Pull request by ColinMaudry**
Closes #917

### CODE REVIEW

**Review by jpmckinney — COMMENTED** [review]
See my earlier comment and suggested edit at https://github.com/open-contracting/standard/pull/1066#pullrequestreview-486923357

File: docs/schema/identifiers.md [hunk 1]
```diff
@@ -87,6 +87,8 @@ The ocid prefix itself is made up of two parts: a prefix agency identifier (curr
 
 The ocid is case-sensitive; in other words, the letter case of an ocid must be consistent.
 
+It is good practice to separate the ocds prefix and the internal identifier with an hyphen (`-`).
```
Comment (jpmckinney) [inline docs/schema/identifiers.md]:
```suggestion
It is encouraged to separate the ocds prefix and the internal identifier with a hyphen (`-`).
```

File: schema/dereferenced-release-schema.json [hunk 2]
```diff
@@ -7,7 +7,7 @@
   "properties": {
     "ocid": {
       "title": "Open Contracting ID",
-      "description": "A globally unique identifier for this Open Contracting Process. Composed of an ocid prefix and an identifier for the contracting process. For more information see the [Open Contracting Identifier guidance](https://standard.open-contracting.org/{{version}}/{{lang}}/schema/identifiers/)",
+      "description": "A globally unique identifier for this Open Contracting Process. Composed of an ocid prefix and an identifier for the contracting process. It is encouraged to separate the ocds prefix and the internal identifier with an hyphen (`-`). For more information see the [Open Contracting Identifier guidance](https://standard.open-contracting.org/{{version}}/{{lang}}/schema/identifiers/)",
```
Comment (jpmckinney) [inline schema/dereferenced-release-schema.json]:
```suggestion
      "description": "A globally unique identifier for this Open Contracting Process. Composed of an ocid prefix and an identifier for the contracting process. It is encouraged to separate the ocds prefix and the internal identifier with a hyphen (`-`). For more information see the [Open Contracting Identifier guidance](https://standard.open-contracting.org/{{version}}/{{lang}}/schema/identifiers/)",
```

File: schema/release-schema.json [hunk 3]
```diff
@@ -7,7 +7,7 @@
   "properties": {
     "ocid": {
       "title": "Open Contracting ID",
-      "description": "A globally unique identifier for this Open Contracting Process. Composed of an ocid prefix and an identifier for the contracting process. For more information see the [Open Contracting Identifier guidance](https://standard.open-contracting.org/{{version}}/{{lang}}/schema/identifiers/)",
+      "description": "A globally unique identifier for this Open Contracting Process. Composed of an ocid prefix and an identifier for the contracting process. It is encouraged to separate the ocds prefix and the internal identifier with an hyphen (`-`). For more information see the [Open Contracting Identifier guidance](https://standard.open-contracting.org/{{version}}/{{lang}}/schema/identifiers/)",
```
Comment (jpmckinney) [inline schema/release-schema.json]:
```suggestion
      "description": "A globally unique identifier for this Open Contracting Process. Composed of an ocid prefix and an identifier for the contracting process. It is encouraged to separate the ocds prefix and the internal identifier with a hyphen (`-`). For more information see the [Open Contracting Identifier guidance](https://standard.open-contracting.org/{{version}}/{{lang}}/schema/identifiers/)",
```

File: docs/schema/identifiers.md (hunk 1 above)
Comment (ColinMaudry) [reply docs/schema/identifiers.md]: Sorry, I had forgotten the documentation.

**Pull request merged**